### PR TITLE
testing stuff

### PR DESCRIPTION
**Pull request by jemten**
### This PR adds | fixes:

- No the star recipe will say hello. Important! Take a look at this @projectoriented 

### How to test:

- Automatic and continuous test suite

### Expected outcome:
- [ ] Installation, unit and integration test suite pass

### Review:
- [ ] Code review
- [ ] Tests pass

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes. If applicable record manual test results in PR header
- [ ] **MINOR** - when you add functionality in a backwards compatible manner. If applicable record manual test results in PR header
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by jemten** [comment]
Only showing mei

File: lib/MIP/Recipes/Analysis/Star_aln.pm [hunk 1]
```diff
@@ -311,6 +311,8 @@ sub analysis_star_aln {
     );
     say {$filehandle} $NEWLINE;
 
+    say STDERR q{Hello};
+
```
Comment (projectoriented) [inline lib/MIP/Recipes/Analysis/Star_aln.pm:315]:
```suggestion
    say STDERR q{Hi};

```

**Review by projectoriented — COMMENTED** [review]
Anders, this.

File: lib/MIP/Recipes/Analysis/Star_aln.pm (hunk 1 above)
Comment (jemten) [reply lib/MIP/Recipes/Analysis/Star_aln.pm:315]: you sure?!

**Pull request closed without merging**